src/main/java/com/gd/clinic/entity/Event.java

```diff
@@ -0,0 +1,40 @@
+package com.gd.clinic.entity;
+
+import com.gd.clinic.model.NewEventDto;
+import lombok.*;
+import org.hibernate.annotations.GenericGenerator;
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+import java.time.OffsetDateTime;
+import java.util.UUID;
+
+@NoArgsConstructor
+@RequiredArgsConstructor
+@Getter
+@Setter
+@ToString
+@Entity
+@Table(name = "events")
+public class Event {
```

Comment:
you need to add implementations of `equals` and `hashCode` methods

Reply:
Done, all entities now override those methods this commit.